### PR TITLE
docs(en): add doubly linked list

### PR DESCRIPTION
**Pull request by Ayebilla**
**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [x] My addition is on refer on the language README.md file
- [x] My addition does not have a spelling problem
- [x] My submission has a proper and user-friendly description of the algorithm
- [x] My submission has the time complexity of the algorithm
- [ ] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [x] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**
Add docs(en) for doubly linked list. 

**Other information:**
 Addresses #docs(en): doubly-linked-list
Please, review.

### CODE REVIEW

File: docs/en/Lists/doubly-linked-list.md
Comment (ming-tsai) [inline docs/en/Lists/doubly-linked-list.md:68]: Do you have a link for this reference?

File: docs/en/Lists/doubly-linked-list.md
Comment (ming-tsai) [inline docs/en/Lists/doubly-linked-list.md:6]: The icon doesn't work, and please remove it.

File: docs/en/Lists/doubly-linked-list.md
Comment (ming-tsai) [inline docs/en/Lists/doubly-linked-list.md:3]: Could you add a new line? due to someone doesn't wrap the word on his editor.

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @Ayebilla,
Some suggestions for this pull request.

File: docs/en/Lists/doubly-linked-list.md
Comment (Ayebilla) [reply docs/en/Lists/doubly-linked-list.md]: Thank you for the suggestions, I have added the link now.

**Comment by Ayebilla** [comment]
The icon has been removed as well. That edit was mistakenly wrapped in the first commit

**Comment by Ayebilla** [comment]
Thank you @ming-tsai for taking time to review my pull request. I have implemented the suggested changes and waiting for your approval.

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**
